uitools/cpp/Esri/ArcGISRuntime/Toolkit/Internal/GeoViews.h

```diff
@@ -52,4 +46,30 @@ namespace Toolkit
 
 #endif
 
+namespace Esri {
+namespace ArcGISRuntime {
+namespace Toolkit {
+  inline Map* getGeoModel(MapViewToolkit* mapView)
```

Comment:
if we're not actually returning the base `GeoModel` should these just be named `getMap`?

Reply:
These are part of a way of giving a `getGeoModel` method to GeoViews as used here:

https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/b6c0d18982a4faef43f7b368cbc5a329f93924dc/uitools/cpp/Esri/ArcGISRuntime/Toolkit/FloorFilterController.cpp#L95

So `getGeoModel` allows for one to be agnostic with the passed type.